lib/samples/add_feature_layers/add_feature_layers_sample.dart

```diff
@@ -60,13 +78,17 @@ class _AddFeatureLayersSampleState extends State<AddFeatureLayersSample> {
   Widget build(BuildContext context) {
     return Scaffold(
       body: SafeArea(
+        top: false,
+        // create a column with a map view and a dropdown button.
         child: Column(
           children: [
+            // add a map view to the widget tree and set a controller.
             Expanded(
               child: ArcGISMapView(
                 controllerProvider: () => _mapViewController,
               ),
             ),
+            // create a dropdown button to select a feature layer source.
             DropdownButton<String>(
```

Comment:
I think here it can just be `DropdownButton(`, and let the compiler do the work.